packages/create-next-app/templates/default/ts/next-rspack.config.ts

```diff
@@ -0,0 +1,9 @@
+import type { NextConfig } from "next";
+import withRspack from "next-rspack";
+
+const nextConfig: NextConfig = {
+  /* config options here */
+  reactStrictMode: true,
+};
+
+export default withRspack(nextConfig);
```

Comment:
Instead of adding this in every template how about we just augment the `next.config` file similar to the `tsconfig.json` updating we do here https://github.com/vercel/next.js/blob/509cf9befaa24f8c5bbd9cff4156dafbf18938b9/packages/create-next-app/templates/index.ts#L84

Reply:
Oh! this is better. I will modify it this way.